compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/tree/STReAtomCharOrEscapeNode.java

```diff
@@ -0,0 +1,84 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
I think license headers of [b5a484d](https://github.com/ballerina-platform/ballerina-lang/pull/37581/commits/b5a484d5ee1fcb2bd94dd2288e191b8a27d42a1f) are incorrect